src/sentry/api/endpoints/group_hashes.py

```diff
@@ -42,3 +45,26 @@ def get(self, request, group):
             order_by='id',
             on_results=lambda x: serialize(x, request.user),
         )
+
+    def delete(self, request, group):
+        id_list = request.GET.getlist('id')
+        if id_list is None:
+            return Response()
+
+        hash_list = GroupHash.objects.filter(
```

Comment:
this is a general quesiton, not suggesting it be in this PR: this might be planned, but does it make sense to add a status to this to indicate its in progress (so the UI can reflect that)